test

```diff
@@ -398,6 +398,42 @@ function release_pass {
 	mv /tmp/etcd ./bin/etcd-last-release
 }
 
+function release_tests_pass {
+
+	if [ -z "${VERSION:-}" ]; then
+		VERSION="3.4.99"
+  	fi
```

Comment:
Can we keep the original functionality from the main, and override VERSION in the `Makefile`, doing:

```sh
test-release:
	PASSES="release_tests" ./test $(GO_TEST_FLAGS) VERSION=3.4.99
```